Add a top level page document our Kubernetes choices (#692)
replicatedhq/help-center: content/embedded-kubernetes/_index.md

                        </li>
                    </ul>
                </div>
            </div>
        </div>
    </div>
</div>

<div class="scheduler-type main-section u-borderBottom--gray">
    <div class="container">
        <div class="paddingContainer">
            <div class='u-flexTabletReflow'>
                <div class='flex1 flex-column flex-verticalCenter u-lineHeight--normal left-content'>
                    <span class="logo replicatedLogo"></span>
                    <h5>Rook and Ceph Storage</h5>
                    <p>The Replicated Embedded Kubernetes cluster is installed with Rook and Ceph, and the installer manages the state of the system. </p>
                    <div class="u-marginTop--small">
                        <a href="/guides/native-scheduler" class="Button secondary">Get Started</a>
                    </div>
                </div>
                <div class='flex1 flex-column u-paddingLeft--most flex-verticalCenter right-content'>
                    <ul class="pros-list">
                        <li class="u-commandPromptRed">
                            <h6>Highly available storage</h6>

>                             <p>Rook provisions and operates a Ceph cluster to manage storage, a battle tested, mature storage system.</p>

Reviewer: ```suggestion
                            <p>Rook provisions and operates Ceph, a battle tested, mature system for distributed fault-tolerant storage.</p>
```

                        </li>
                        <li class="u-dockerRed">
                            <h6>Relocatable Volumes</h6>

>                             <p>Automatic support for persistent volumes, which enables Kubernetes to reschedule Pods anywhere in the cluster, and the data will stioll be available.</p>

Reviewer: typo

```suggestion
                            <p>Automatic support for persistent volumes, which enables Kubernetes to reschedule Pods anywhere in the cluster while ensuring data moves with your workloads.</p>
```

                        </li>
                        <li class="u-containersRed">
                            <h6>Block and Blob Store</h6>
                            <p>Rook provides both block store as a drop-in replacement for standard HostPath volumes, and blob store to implement the S3 protocol on any cluster.</p>
                        </li>
                    </ul>
                </div>
            </div>
        </div>
    </div>
</div>

<div class="scheduler-type main-section u-borderBottom--gray">
    <div class="container">
        <div class="paddingContainer">
            <div class='u-flexTabletReflow'>
                <div class='flex1 flex-column flex-verticalCenter u-lineHeight--normal left-content'>
                    <span class="logo dockerSwarmLogo-combo"></span>
                    <h5>Weave Networking</h5>

>                     <p>Every Replicated Embedded Kubernetes cluster comes with Weave overlay networking configured and CoreDNS.</p>

Reviewer: ```suggestion
                    <p>Every Replicated Embedded Kubernetes cluster comes with secure-by-default Weave overlay networking for inter-container communication</p>
```

                    <div class="u-marginTop--small">
                        <a href="/guides/ship-with-docker-swarm/" class="Button secondary">Get Started</a>
                    </div>
                </div>
                <div class='flex1 flex-column u-paddingLeft--most flex-verticalCenter right-content'>
                    <ul class="pros-list">
                        <li class="u-commandPromptBlue">
                            <h6>CoreDNS Support</h6>

>                             <p>By default, Kubernetes will register service and pods with CoreDNS, allowing for container to container communication on the overlay network.</p>

Reviewer: ```suggestion
                            <p>By default, Kubernetes will register service and pods with CoreDNS, allowing for state-of-the art service discovery via DNS lookups.</p>
```

                        </li>
                        <li class="u-ymlBlueLight">

>                             <h6>Easier On Firewalls</h6>

Reviewer: Don't have an alternative, but this heading feels a little off tone-wise

                        </li>
                    </ul>
                </div>
            </div>
        </div>
    </div>
</div>

<div class="scheduler-type main-section u-borderBottom--gray">
    <div class="container">
        <div class="paddingContainer">
            <div class='u-flexTabletReflow'>
                <div class='flex1 flex-column flex-verticalCenter u-lineHeight--normal left-content'>
                    <span class="logo dockerSwarmLogo-combo"></span>
                    <h5>Contour Ingress</h5>
                    <p>Contour is a portable ingress controller from Heptio.</p>
                    <div class="u-marginTop--small">
                        <a href="/guides/ship-with-docker-swarm/" class="Button secondary">Get Started</a>
                    </div>
                </div>
                <div class='flex1 flex-column u-paddingLeft--most flex-verticalCenter right-content'>
                    <ul class="pros-list">
                        <li class="u-commandPromptBlue">
                            <h6>Portable</h6>

>                             <p>The contour ingress controller is compatible anywhere the cluster is installed; cloud providers and care metal incldued.</p>

Reviewer: ```suggestion
                            <p>The contour ingress controller is compatible anywhere the cluster is installed; cloud providers and bare metal incldued.</p>
```

>                         </li>

Reviewer: I'd add another few headers here:

- Built on Envoy
    - Built on the mature, battle-tested Envoy proxy 
- Cloud native, on prem
    - Instant reconfiguration, dynamic ingress routes, and native WebSocket support
- The future of Ingress
    - Leverage next-generation Ingress features like route inheritance and upstream health checking

                    <div class="u-marginTop--small">
                        <a href="/guides/ship-with-docker-swarm/" class="Button secondary">Get Started</a>
                    </div>
                </div>
            </div>
        </div>
    </div>
</div>

<div class="scheduler-type main-section u-borderBottom--gray">
    <div class="container">
        <div class="paddingContainer">
            <div class='u-flexTabletReflow'>
                <div class='flex1 flex-column flex-verticalCenter u-lineHeight--normal left-content'>
                    <span class="logo heptioContour-logo"></span>
                    <h5>Contour Ingress</h5>
                    <p>Contour is a portable ingress controller from Heptio.</p>
                    <div class="u-marginTop--small">
                        <a href="/guides/ship-with-docker-swarm/" class="Button secondary">Get Started</a>
                    </div>
                </div>
                <div class='flex1 flex-column u-paddingLeft--most flex-verticalCenter right-content'>
                    <ul class="pros-list">
                        <li class="u-envoyLogoBlue">
                            <h6>Built On Envoy</h6>

>                             <p>Built on the mature, battle-tested Envoy proxy that powers dynamic infrastructure at massive scale.</p>

Reviewer 2: "mature battle tested" twice on the same page, makes it look like just buzz words
